server/sonar-web/src/main/js/apps/projects/components/ProjectCard.js

```diff
@@ -99,6 +89,31 @@ export default function ProjectCard({ measures, organization, project, type }: P
           <Link to={{ pathname: '/dashboard', query: { id: project.key } }}>{project.name}</Link>
         </h2>
         {displayQualityGate && <ProjectCardQualityGate status={measures['alert_status']} />}
+        <div className="pull-right text-right">
+          {isPrivate && <PrivateBadge className="spacer-left" tooltipPlacement="left" />}
+          {hasTags && <TagsList tags={project.tags} customClass="spacer-left" />}
+        </div>
+        {isLeakView &&
+          isProjectAnalyzed &&
+          <div
+            className={classNames('project-card-dates note text-right pull-right', {
+              'width-100': isPrivate || hasTags
+            })}>
+            {hasLeakPeriodStart &&
+              <span>
+                {translateWithParameters(
+                  'projects.leak_period_x',
+                  moment(project.leakPeriodDate).fromNow()
+                )}
+              </span>}
+            {isProjectAnalyzed &&
```

Comment:
this block is included in `{isLeakView &&`, so you don't see the analysis date in the "Overall" mode